autoware_lanelet2_extension/include/autoware_lanelet2_extension/regulatory_elements/bus_stop_area.hpp

```diff
@@ -0,0 +1,77 @@
+// Copyright 2024 Tier IV, Inc.
```

Comment:
I think it's better in this repos.

```suggestion
// Copyright 2024 Autoware Foundation. All rights reserved.
```

Reply:
fixed in 61ed64f2e6a09cb15e5d4f542207e03514f792b1

Reply:
It still remains here.

Reply:
Sorry I was fixing the same line in cpp file. should have been fixed in the next commit.

I'm adding unit-test now.